Newsletters: add 184 (2022-01-26)

harding opened the pull request:
- [x] Releases, RCs, topic links @harding
- [x] Bitcoin Stack Exchange @bitschmidty 
- [x] Eclair 2134 Activate anchor outputs @dongcarl 
- [x] BIPs #1270 BIP-174 Clarify that partial_sig should be a valid @Xekyo

harding commented:
Quick skim of @bitschmidty's SE section and @dongcarl's and @Xekyo's PR descriptions LGTM.  Thanks y'all!

Xekyo commented on _posts/en/newsletters/2022-01-26-newsletter.md:
```diff
@@ -0,0 +1,148 @@
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Is it possible to convert a taproot address into a native segwit address?]({{bse}}111440)
+  After an exchange changed a user's segwit v1 P2TR taproot withdrawal address
+  into a segwit v0 native segwit P2WSH address due to lack of taproot support,
```
Both P2TR and P2WSH are native segwit addresses. Also, "segwit v0 native segwit" feels a bit off. Perhaps this would be better:

```suggestion
  After an exchange changed a user's P2TR (native segwit v1) taproot withdrawal address
  into a P2WSH (native segwit v0) address due to lack of taproot support,
```

Xekyo commented on _posts/en/newsletters/2022-01-26-newsletter.md:
```diff
@@ -0,0 +1,148 @@
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Eclair #2134][] enables [anchor outputs][topic anchor outputs] by default,
+  allowing commitment transactions to be fee bumped should its feerate be too
```
Plural subject with singular possessive pronoun. Perhaps one of these:

```suggestion
  allowing a commitment transaction to be fee bumped should its feerate be too
```

```suggestion
  allowing commitment transactions to be fee bumped should their feerate be too
```

Xekyo commented on _posts/en/newsletters/2022-01-26-newsletter.md:
```diff
@@ -0,0 +1,148 @@
+  includes classifying transactions by the importance of confirming them
+  on time, re-evaluating transactions after each block to determine
+  whether it's appropriate to fee bump them, also re-evaluating current
+  network feerates to in case the transaction's feerate needs to be
```
Methinks that "to" is a stray

```suggestion
  network feerates in case the transaction's feerate needs to be
```

Xekyo commented on _posts/en/newsletters/2022-01-26-newsletter.md:
```diff
@@ -0,0 +1,148 @@
+
+- [BTCPay Server #3083][] allows administrators to log into a BTCPay
+  instance using [LNURL authentication][] (which is also possible to
+  implement in non-LN software).
```
Perhaps:
```suggestion
 be implemented in non-LN software).
```

Xekyo commented on _topics/en/covenants.md:
```diff
@@ -75,9 +75,15 @@ see_also:
     link: topic vaults
 ---
+> "In the context of Bitcoin, the most useful definition of
+> covenant is that it's when the scriptPubKey of a UTXO restricts the
+> scriptPubKey in the output(s) of a tx spending that utxo."  ---Anthony
```
I assume that's a direct quote, but it's a bit odd that UTXO is capitalized once and lowercased once.

```suggestion
> covenant is that it's when the scriptPubKey of a UTXO restricts the
> scriptPubKey in the output(s) of a tx spending that UTXO."  ---Anthony
```

harding replied:
Whoops, my fault, not AJs.  I uppercased one instance and missed the second.  Fixed, thanks!

Xekyo commented on _topics/en/pay-to-contract-outputs.md:
```diff
@@ -0,0 +1,66 @@
+    url: /en/newsletters/2021/10/20/#pay-to-contract
+    date: 1970-01-01
+
+  - title: Proposal to add extension fields to PSBTS for pay-to-contract
```
To get the plural of an acronym I'd expect the addition of a lowercase "s":

```suggestion
  - title: Proposal to add extension fields to PSBTs for pay-to-contract
```

Xekyo reviewed (commented):
Nice selection from Stack Exchange, and that's quite a few notable code changes! Thanks for always being on top of that.

bitschmidty commented on _topics/en/pay-to-contract-outputs.md:
```diff
@@ -0,0 +1,66 @@
+optech_mentions:
+  - title: "Taproot history: pay to contract protocol"
+    url: /en/newsletters/2021/10/20/#pay-to-contract
+    date: 1970-01-01
```
I believe this date is part of the boilerplate new topic text that can be removed.

bitschmidty commented on _topics/en/pay-to-contract-outputs.md:
```diff
@@ -0,0 +1,66 @@
+  - title: Proposal to add extension fields to PSBTS for pay-to-contract
+    url: /en/newsletters/2019/03/12/#extension-fields-to-partially-signed-bitcoin-transactions-psbts
+
+  - title: Eclair #965 allows specifying custom preimages, which can be used with pay-to-contract
```
The "#" in this title requires the title to be quoted in order for the title to display properly.

glozow commented on _posts/en/newsletters/2022-01-26-newsletter.md:
```diff
@@ -0,0 +1,148 @@
+  Murch explains that ECDSA signature grinding is the process of repeatedly
+  signing until you get a signature whose r-value is in the lower half of the
+  range, resulting in a signature that is 1 byte smaller (32 bytes vs 33 bytes)
+  based on ECDSA's serialization format. This smaller signature results in less
```
```suggestion
  based on ECDSA's serialization format. This smaller signature results in lower
```

glozow commented on _topics/en/pay-to-contract-outputs.md:
```diff
@@ -0,0 +1,66 @@
+## Should be less than 500 characters
+excerpt: >
+  **Pay-to-contract protocols** allow a spender and a receiver to agree
+  on the text of a contract (or anything else) and then create an
```
```suggestion
  on the text of a contract (or anything else) and then create a
```

glozow reviewed (commented):
lgtm, great stack exchange selection :ok_hand:

harding commented:
Made suggested edits (thanks so much @Xekyo, @bitschmidty, and @glozow !) and added topics.  I don't see any releases or RCs this week, so I dropped that section.

Thank you everyone!

bitschmidty commented:
Thanks for your contributions this week, @harding @dongcarl @Xekyo @glozow !
